.github/workflows/monorepo-ci.yml

```
@@ -31,8 +35,120 @@
   contents: read
   checks: write
   pull-requests: write
+  security-events: write # Required for CodeQL to upload results
+
+labeler:
```

actionlint (1.7.7)

unexpected key "labeler" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)